server/index.js

```diff
@@ -2,9 +2,14 @@ const express = require('express');
 
 const app = express();
 const port = 3003;
+const db = require('../data/index.js');
```

Comment:
@VALE46GP There doesn't appear to be anything exported from this file. Additionally, is there a better name you could use to describe the object that should be exported from `../data/index.js`? IE: Are you directly interacting with the database or are you interacting with a model?